wip-trampoline.md

```diff
@@ -63,36 +43,67 @@ Each `trampoline_hop_payload` has the following structure:
 1. type: `trampoline_hop_payload`
 2. data:
    * [`1`:`num_frames_and_realm`]
-   * [`1`:`type`]
-   * [`1`:`length`]
+   * [`47`:`next_trampoline_node`]
```

Comment:
Like mentioned before `next_trampoline_node` should itself be expressed as TLV-pairs in a single TLV, in case we want to switch out the way we address the nodes, allowing us to reduce the footprint.

Reply:
I'll explore that option.